image_width: 1280
image_height: 720
camera_name: sensor_measurement/camera
camera_matrix:
  rows: 3
  cols: 3
  data: [956.5530, 0, 639.9951, 0, 955.6364, 348.6131, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.0078,-0.1778, -0.0012, -0.0016, 0.4409]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [956.5530, 0, 639.9951,0, 0, 955.6364, 348.6131,0, 0, 0, 1, 0]